tests/operators/__init__.py

```diff
@@ -7,21 +7,26 @@
 # to you under the Apache License, Version 2.0 (the
 # "License"); you may not use this file except in compliance
 # with the License.  You may obtain a copy of the License at
-# 
+#
 #   http://www.apache.org/licenses/LICENSE-2.0
-# 
+#
 # Unless required by applicable law or agreed to in writing,
 # software distributed under the License is distributed on an
 # "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
 # KIND, either express or implied.  See the License for the
 # specific language governing permissions and limitations
 # under the License.
-
+from subprocess import check_call
 from .docker_operator import *
 from .subdag_operator import *
 from .operators import *
 from .hive_operator import *
-from .s3_to_hive_operator import *
+try:
+    check_call(["kubectl", "get", "pods"])
+except Exception as e:
+    print(e)
+    from .s3_to_hive_operator import *
+    pass
```

Comment:
This looks like debugging code?

Reply:
Here's an example build where it fails because it's attempting to run the moto decorator even though it shouldn't be able to https://travis-ci.org/bloomberg/airflow/jobs/423162910#L4949